### ISSUE TITLE
How to hide a function definition

### ISSUE THREAD

**Issue by v1d3rm3** [starbelly/rebar3_ex_doc#17]
Is it EDoc compliant?
I decorate function with `@hidden`, but it stills appear in ex_doc page.

**Comment by starbelly** [comment]
@v1d3rm3 For ex_doc, `@private` will do the trick. `@hidden` is specific to edoc I believe.

**Comment by starbelly** [comment]
I'm going to close this as resolved. Feel free to re-open if you feel there's a problem.

**Comment by eproxus** [comment]
I have the same problem. Neither `@hidden` nor `@private` works.

E.g:

```erl

-export([hidden/0]).
-export([private/0]).

% @hidden
hidden() -> ok.

% @private
private() -> ok.
```

**Comment by starbelly** [comment]
Yeah, seems to not be working now. We'll have to look into ex_doc as to what the issue is. Need to verify it doesn't happen when running ex_doc on its own.

**Comment by starbelly** [comment]
@eproxus This seems to be related to OTP 25. I just published rebar3_hex and it everything was exposed, I then switched to OTP 24 and republished : https://hexdocs.pm/rebar3_hex/7.0.2/api-reference.html#modules

This will take some digging.

**Comment by eproxus** [comment]
@starbelly Thanks for looking into it! I’ll use that as a workaround for now 👍

**Comment by starbelly** [comment]
@eproxus  This was fixed in #30 and 0.2.12 cut and pushed to hex https://hex.pm/packages/rebar3_ex_doc/0.2.12

**Comment by eproxus** [comment]
Lovely! 🙏
